internal/rukpak/source/helpers.go

```diff
@@ -0,0 +1,80 @@
+package source
```

Comment:
So this one is the same code of old [internal/rukpak/source/util.go](https://github.com/operator-framework/operator-controller/pull/1677/files#diff-fefb4b08659747f09c820f25987bf53a887fb543dfb23b57b1eaeeaa5f797ee9), but it's been cleaned up and refactored for better readability and maintainability. 👍

Reply:
yeah - gh is screwing up the diff. If you [here](https://github.com/operator-framework/operator-controller/pull/1677/commits/0d4d124b121e34071be5165696663639847223dd#diff-df5f93b531287a6385e6972872b1c33241e5265d99a058e0f8086c95e57e4d7b) it gives you the proper diff =S